tools/niminst/niminst.nim

```diff
@@ -513,7 +513,10 @@ template gatherFiles(fun, libpath, outDir) =
       fun(src, dst)
 
     for f in walkFiles(libpath / "lib/*.h"): copySrc(f)
-    copySrc(libpath / "lib/wrappers/linenoise/linenoise.h")
```

Comment:
Seriously? This is your fix? "let's always copy linenoise.h except if it fails we don't mind"?